samples/annotate/project-build.xml

```diff
@@ -134,13 +134,12 @@
 		<mkdir dir="${basedir}/target/test-classes"/>
 		<javac
 			destdir="${basedir}/target/test-classes" 
-			srcdir="${basedir}/src/test/java:${basedir}/target/generated-sources/xjc"
```

Comment:
I've tried to run full build: `mvn clean install -DperformRelease -Psamples,tests`
but ant-task fails, because this dir is absent.. well, maybe I misunderstood how to use it correctly